client/src/Components/LoginForm/index.js

```diff
@@ -0,0 +1,80 @@
+import React, { Fragment, Component } from 'react';
+import { TextField } from '@material-ui/core';
+
+class TextFieldValidation extends Component {
+	state = {};
+	render() {
+		let { onChange, error } = this.props;
+		return (
+			<Fragment>
```

Comment:
Good use of Fragment 👍 

Relatively recently facebook added support for Fragments to be written like `<>...</>` which skips the need to import, but nothing wring with using the more explicit component.